config/environments/production.rb

```diff
@@ -37,7 +37,12 @@
   # config.active_storage.service = :local
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
+  config.ssl_options = {
+    redirect: {
+      exclude: ->(request) { !ENV['TLS_CA_PATH'] }
+    }
+  }
```

Comment:
Something like this should make sure we're only forcing ssl when the `TLS_CA_PATH` is set(when TLS is being used)
Still need to set up the CA cert though.

Reply:
Again, please make sure you know which cert is which.